tests/requirements/sanic-20.txt

```diff
@@ -0,0 +1,7 @@
+# these are constrained by the files in tests/constraints/*.txt
```

Comment:
sanic 20.* needs a separate requirements file to remove the `sanic-testing` requirement.